pkg/volume/cephfs/cephfs.go

```diff
@@ -207,10 +209,17 @@ func (cephfsVolume *cephfsMounter) GetAttributes() volume.Attributes {
 	}
 }
 
-// Checks prior to mount operations to verify that the required components (binaries, etc.)
+// CanMount checks prior to mount operations to verify that the required components (binaries, etc.)
 // to mount the volume are available on the underlying node.
 // If not, it returns an error
 func (cephfsMounter *cephfsMounter) CanMount() error {
+	exe := cephfsMounter.plugin.host.GetExec(cephfsMounter.plugin.GetPluginName())
```

Comment:
No need to change this method, since we assume binary `mount` is present.

Reply:
Got it, thanks!